### ISSUE TITLE
Multi-All-Day-Meetings

### ISSUE THREAD

**Issue by saerdnaer** [voc/eventkalender#1]
Aktuell werden nur Conferences als mehrtägige Veranstaltungen im Kalender eingetragen, Meetings sind dagegen wohl immer Einzeltermine mit Start- und Endzeit, vgl: 
<img width="449" alt="" src="https://cloud.githubusercontent.com/assets/40266/13900003/fb3f92e8-edfb-11e5-9b44-fe13aebe6395.png">

Das Media Geekend sollte aus meiner Sicht eigentlich ein All-day-event vom 2016-04-23 bis zum 2016-04-24 sein und kein 24h Termin von 2016-04-23 0:00 bis 2016-04-24 0:00. Sprich da muss ne extra Fallunterscheidung für ein Datum ohne Uhrzeit (vgl. z.B. https://github.com/muccc/wiki_kalenderscraper/commit/ee3e3fc83c6f3b5522b45d13e59e002b889b0243#diff-4d43e421853d4a6f9864797ccbe89199R55 ) mit rein.

**Comment by meise** [comment]
Änderung ist deployed. Bitte mal prüfen ob die Zeiträume jetzt passen.

**Comment by saerdnaer** [comment]
Nein, leider noch nicht. Gelb im Screenshot ist so wie es aktuell ist – blau so wie es eigentlich sein sollte.
![image](https://cloud.githubusercontent.com/assets/40266/24110643/4051b15a-0d94-11e7-9b1e-728836e12ee2.png)
Mehr Details warum das aktuell falsch dargestellt wird in https://github.com/voc/eventkalender/commit/74b8b83f36fb1648b4751183c5a20ad58a207b03#comments
An sich kann es die iCal Biblothek ja (vgl. Konferenzen), man darf ihr wohl halt dem date halt nur kein 0:00 als Uhrzeit mitgeben.
